Extension/src/LanguageServer/client.ts

```diff
@@ -1550,8 +1544,8 @@
         });
     }
 
-    public updateCustomBrowseConfiguration(requestingProvider?: CustomConfigurationProvider1): Thenable<void> {
-        return this.notifyWhenReady(() => {
+    public async updateCustomBrowseConfiguration(requestingProvider?: CustomConfigurationProvider1): Promise<void> {
+        return this.notifyWhenReady(async () => {
```

Comment:
it doesn't look like there was any need to make this or the line above `async`.  Can you revert it?